modules/friendlytag.js

```diff
@@ -46,35 +46,38 @@ Twinkle.tag.callback = function friendlytagCallback() {
 	var form = new Morebits.quickForm(Twinkle.tag.callback.evaluate);
 
 	// if page is unreviewed, add a checkbox to the form so that user can pick whether or not to review it
-	new mw.Api().get({
-		action: 'pagetriagelist',
-		format: 'json',
-		page_id: mw.config.get('wgArticleId')
-	}).done(function(response) {
-		var isReviewed = false;
-		var isOldPage = response.pagetriagelist.result !== 'success' || response.pagetriagelist.pages.length === 0;
-		var isMarkedAsReviewed = response.pagetriagelist.pages[0].patrol_status > 0;
-		if (isOldPage || isMarkedAsReviewed) {
-			isReviewed = true;
-		}
+	let isPatroller = mw.config.get('wgUserGroups').some(r => ['patroller', 'sysop'].includes(r));
+	if (isPatroller) {
+		new mw.Api().get({
+			action: 'pagetriagelist',
+			format: 'json',
+			page_id: mw.config.get('wgArticleId')
+		}).done(function(response) {
```

Comment:
`.done` can be changed to `.then` which is more modern and compatible with ES6 promise spec.